portfolio/src/main/java/com/google/sps/servlets/BlogCommentServlet.java

```diff
@@ -152,5 +162,53 @@ private String getUploadedFileUrl(HttpServletRequest request, String formInputEl
       return imagesService.getServingUrl(options);
     }
   }
+
+  /* returns the sentiment score of the message */
+  private float getSentimentScore(String message) {
+    
+    try {
+      Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+      LanguageServiceClient languageService = LanguageServiceClient.create();
+      
+      Sentiment sentiment = languageService.analyzeSentiment(doc).getDocumentSentiment();
+      float score = sentiment.getScore();
+      languageService.close();
+      return score;
+    }
+    catch(IOException e) {
+      e.printStackTrace();
+    }
+    return 0;
+  }
+
+  /* returns the content classifications of the message */
+  private String classifyContent(String message) {
+    
+    try {
+      // content classification only works for 20+ words
+      int length = message.split(" ").length;
+      if (length > 20) {
+        Document doc = Document.newBuilder().setContent(message).setType(Document.Type.PLAIN_TEXT).build();
+        LanguageServiceClient languageService = LanguageServiceClient.create();
+        
+        ClassifyTextRequest request = ClassifyTextRequest.newBuilder().setDocument(doc).build();
+        // detect categories in the given text
+        ClassifyTextResponse response = languageService.classifyText(request);
+        StringBuilder sb = new StringBuilder();
+        for (ClassificationCategory category : response.getCategoriesList()) {
+            sb.append(
+                "Category name: " + category.getName() + ", Confidence: " + category.getConfidence() + "\n");
+        }
+        return sb.toString();
+      }
+      else {
+        return "";
+      }
```

Comment:
Since you have a catch-all return at the end of the method, you can skip this else clause.